pip_check_reqs/find_extra_reqs.py

```diff
@@ -116,11 +116,11 @@ def main():
 
     logging.basicConfig(format='%(message)s')
     if options.debug:
-        log.setLevel(logging.DEBUG)
```

Comment:
I'm a little confused here and admittedly I am no `logging` expert. IIUC, perhaps this would be made clearer by creating a new variable `global_logger = logging.getLogger()`?

Reply:
And... is that good practice? Could we instead not pass the logging option through to helper files?

Reply:
@adamtheturtle This was easier to do it, but I guess you're right. It might wake up loggers from other imports, which would be confusing e.g. in case of pip. I'll change it.

Reply:
Done. I made it call `setLevel` on both loggers, instead of doing it globally.